ansible/participants-setup.yaml

```diff
@@ -1,36 +1,7 @@
 ---
 - name: Setup Participant
   hosts: local
-  vars:
-    participant_namespace: "{{ participant }}-gitops"
   tasks:
-    - name: "{{ participant_namespace }} namespace is {{ state }}"
```

Comment:
Looks good to me!

Reply:
The linter didn't complain, it just died trying to set itself up 🙃